Elle0

esden opened the pull request:
Adding support for the ELLE0 V1.0 autopilot. This autopilot is still in beta phase, but for easier testing and further integration it would be nice if it was part of the mainline. :)

fvantienen commented on sw/airborne/boards/elle0/baro_board.h:
```diff
@@ -0,0 +1,18 @@
+ *
+ */
+
+#ifndef BOARDS_LISA_MX_BARO_H
```
Forgot to rename something?

fvantienen commented on sw/airborne/boards/elle0/baro_board.h:
```diff
@@ -0,0 +1,18 @@
+
+/*
+ * board specific functions for the lisa_mx board
```
also here...

fvantienen commented on sw/airborne/boards/elle0_1.0.h:
```diff
@@ -0,0 +1,446 @@
+#define ACTUATORS_PWM_NB 6
+#endif
+
+// Servo numbering on LisaM silkscreen/docs starts with 1
```
Lisam?

fvantienen commented on sw/airborne/boards/elle0_1.0.h:
```diff
@@ -0,0 +1,446 @@
+/*
+ * Copyright (C) 2013 Piotr Esden-Tempski <piotr@esden.net>
```
You live in the past :smile:

fvantienen commented:
Cool.. is it already working a bit?

esden commented:
@fvantienen of course otherwise I would not be sending in a pull request ;)

https://www.youtube.com/watch?v=QBs-y-IvI9E

flixr commented:
The board files look good :-)

Regarding the airframe files:
- We said that we would only keep airframes in conf/examples that are actually part of the example of test conf...
- it would make sense to use the correct motor mixing
- not so sure about the value of the setup_elle0 file

esden commented:
Regarding the setup. It would be great to find a solution of compiling the setup firmwares without having to put it into either the airframe file or having a "dummy" airframefiles for the specific autopilot. The best thing would be to have those targets available for every aircraft that uses certain autopilot. But that is a fix that needs to be done separately. I use those setup firmwares every time for debugging and configuration purposes. That is why I like the fact that they are separate "aircraft" a better solution would be nice though...

Regarding your other comments I am working on a patch and I will push it in a few.

hooperfly commented:
@esden maybe the motor layouts for the examples should reflect the ordering @flixr introduced for default motor layouts. In the case of a quad-x configuration.

```
<set servo="TOP_LEFT" value="motor_mixing.commands[MOTOR_FRONT_LEFT]"/>
<set servo="TOP_RIGHT" value="motor_mixing.commands[MOTOR_FRONT_RIGHT]"/>
<set servo="BOTTOM_RIGHT" value="motor_mixing.commands[MOTOR_BACK_RIGHT]"/>
<set servo="BOTTOM_LEFT" value="motor_mixing.commands[MOTOR_BACK_LEFT]"/>
```

esden commented:
The setup I have put into the example airframe file is the one I always had on all of my rotorcraft since forever. That is why I have that in this frame as it matches my test airframe. :) I can change that if this is the desired setup for the example airframes but that also means that I will not be using it on my test airframe...

esden commented:
Ok. I have cleaned up the patches. I added back the ```USE_SERVOS_7AND8``` because we can use it to make them available for the upcomming WS2812 module as well as auxiliary I2C1 interface.
